docs/storage/azure-storage.md

```diff
@@ -58,188 +71,149 @@ You also need to assign the following roles to the user account you are logged i
-    }
-}
+Use the following Razor markup to create a basic form, replacing the contents of the _Home.razor_ file in the _AspireStorage/Components/Pages_ directory:
+
+:::code language="razor" source="snippets/tutorial/AspireStorage/AspireStorage/AspireStorage/Components/Pages/Home.razor":::
+
+:::zone pivot="azurite"
+
+## Update the AppHost
+
+The _AspireStorage.AppHost_ project is the orchestrator for your app. It's responsible for connecting and configuring the different projects and services of your app. The orchestrator should be set as the startup project.
+
+Add the [Aspire.Hosting.Azure](https://www.nuget.org/packages/Aspire.Hosting.Azure) NuGet package to your _AspireStorage.AppHost_ project:
+
+```dotnetcli
+dotnet add package Aspire.Hosting.Azure --prerelease
 ```
 
+Next, replace the contents of the _Program.cs_ file in the _AspireStorage.AppHost_ project with the following code:
+
+:::code source="snippets/tutorial/AspireStorage/AspireStorage.AppHost/Program.cs":::
+
+The preceding code adds Azure storage, blobs and queues, and when in development mode, it uses the emulator.
+
+:::zone-end
+
 ## Process the items in the queue
 
-When a new message is placed on the `tickets` queue, the worker service should retrieve, process, and delete the message. Update the _Worker.cs_ class to match the following code:
+When a new message is placed on the `tickets` queue, the worker service should retrieve, process, and delete the message. Update the _Worker.cs_ class replacing the contents with the following code:
 
-```csharp
-using Azure.Storage.Queues;
-using Azure.Storage.Queues.Models;
-
-namespace AspireProcessor;
-
-public sealed class Worker(
-    QueueServiceClient client,
-    ILogger<Worker> logger) : BackgroundService
-{
-    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
-    {
-        while (!stoppingToken.IsCancellationRequested)
-        {
-            var queueClient = client.GetQueueClient("tickets");
-            QueueMessage[] messages =
-                await queueClient.ReceiveMessagesAsync(maxMessages: 25);
-
-            foreach(var message in messages)
-            {
-                logger.LogInformation(
-                    "Message from queue: {Message}", message.MessageText);
-
-                await queueClient.DeleteMessageAsync(
-                    message.MessageId, message.PopReceipt);
-            }
-        }
-    }
-}
-```
+:::zone pivot="azurite"
+
+:::code source="snippets/tutorial/AspireStorage/AspireStorage.Worker/Worker.cs":::
+
+Before the worker service can process messages, it needs to be able to connect to the Azure Storage queue. With Azurite, you need to ensure that the queue is available before the worker service starts executing messaging queue processing.
+
+:::zone-end
+:::zone pivot="azure-portal,azure-cli"
+
+:::code source="snippets/tutorial/AspireStorage/AspireStorage.Worker/Worker.cs" range="1-11,15-37":::
+
+:::zone-end
 
 ## Run and test the app locally
 
```

Comment:
Minor thing - just realizing we should probably update this section to have them view the worker service logs in the Aspire Dashboard instead of the Visual Studio output window, we could do that later.

Reply:
Good callout, I'll update that